helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -13,5 +13,23 @@ data:
         region: '{{ .Values.Installation.V1.Provider.AWS.Region }}'
       installation:
         name: '{{ .Values.Installation.V1.Name }}'
+        {{- if .Values.Installation.V1.Guest }}
+        guest:
+          {{- if .Values.Installation.V1.Guest.Kubernetes }}
+          kubernetes:
+            {{- if .Values.Installation.V1.Guest.Kubernetes.API }}
+            api:
```

Comment:
To me it looks like we can skip `api:` and `provider:` parts of this already long path. Are there other parts of kubernetes that could require kubernetes?

Reply:
Configmap just reflects structure in draugthsman configmap

Reply:
So all 3 `if` above are not necessary.